charts/featureform/templates/streamer/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+#  Copyright 2024 FeatureForm Inc.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: iceberg-streamer
+spec:
+  {{- if not .Values.icebergStreamer.autoscaling.enabled }}
```

Comment:
Better cluster resource management, don't want to force that on an admin since it's not going to be relevant in all cases either (non-production, etc).